html/janus.js

```diff
@@ -162,7 +191,7 @@ Janus.init = function(options) {
 				}
 			}
 			if(src === 'adapter.js') {
-				if(window.getUserMedia && window.RTCPeerConnection) {
+				if(navigator.getUserMedia && navigator.mediaDevices.getUserMedia && window.RTCPeerConnection) {
 					// Already loaded
```

Comment:
Instead of comment //Already loaded can you please add some Janus.debug log that adapter is already loaded. The same for jquery. It is always nice to have such things for validation purposes especially when loading scripts in some sophisticated order and condition.